app/query_param.go

```diff
@@ -9,10 +9,13 @@ import (
 	"github.com/topfreegames/mqtt-history/logger"
 )
 
-func ParseHistoryQueryParams(c echo.Context, defaultLimit int64) (string, int64, int64, bool) {
+func ParseHistoryQueryParams(c echo.Context, defaultLimit int64) (string, string, string, int64, int64, int64, bool) {
```

Comment:
I was thinking about the changes on this method... as it is used by the other endpoints that serve to the players maybe it could be good to create a copy of it (`ParsePSHistoryQueryParams` ?) instead of modifying it. Does it make sense?

Reply:
Yes, Dani! Makes tottaly sense, thanks!